### DISCUSSION TITLE
How to show speed and convert it?

### DISCUSSION THREAD

**Discussion by sandros87** [gpx-animator/gpx-animator#532] [Q&A]
Hello
I'm trying to show the real time speed. My GPX comes from the app Geo Tracker that saves speed in this format

```
<trkpt lat="44.49297383" lon="11.34936744">
        <ele>101.68</ele>
        <time>2023-05-05T15:37:38Z</time>
        <extensions>
          <geotracker:meta c="2.32" s="4.88" />
        </extensions>
      </trkpt>
```

where "s" is speed in m/s.
So I would like to show this parameter in km/h (converted). Now I'm just getting 0 km/h so that value is not considered nor it's calculated automatically based on timestamps.

**Comment by McPringle** [comment]
Sorry for my late reply, the last few months I took a sabbatical and rode my bicycle through the whole US. I hope you did not stop using GPX Animator in the meantime!

GPX Animator does not pick up the speed tracked by your GEO Tracker because it uses his own extension, as you can see in the XML file. GPX Animator uses the standard GPX file format specification for the speed value: https://www.topografix.com/gpx_manual.asp#speed

If your GEO Tracker does not support the GPX standard for the speed value, you might contact them with a feature request. Being standard compliant is always a good idea. Another possible solution could be to write a plugin for GPX Animator to support the GEO Tracker extensions. But for this to work, the plugin interface needs to be enhanced first. Or you find a solution to convert your file. Maybe GPSBabel can do this?
